src/javascript/app/pages/form/form-component.jsx

```diff
@@ -297,7 +297,7 @@ export const FormComponent = () => {
                                             label={localize('Last Digit Prediction')}
                                             start={0}
                                             end={9}
-                                            elementId='prediction'
+                                            element_id='prediction'
```

Comment:
`elementId` to `element_id` in `NumbersDropdown` as well.

Reply:
same for this I kept it `elementId`

Reply:
@prince-deriv I guess you missed here.